Phonebook.Frontend/src/i18n/messages.de.xlf

```diff
@@ -1238,6 +1242,51 @@ Diese Seite nutzt Cookies, um die Bedienung der Seite zu verbessern. Cookies wer
         <note priority="1" from="meaning">UserDetailComponent</note>
+      </trans-unit><trans-unit id="Headline" datatype="html">
+        <source>Incorrect User Information</source><target state="new">Benutzerinformation</target>
+        <note priority="1" from="description">Headline</note>
+        <note priority="1" from="meaning">User-InformationDialog</note>
+      </trans-unit><trans-unit id="Your Profile" datatype="html">
+        <source>
+      Dear <x id="INTERPOLATION" equiv-text="{{ data.person.Firstname }}"/>,<x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+      unfortunately, you can't change your user information via the Phonebook interface. <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+      If your data is incorrect, please contact your Human Resources department.
+    </source><target state="new">
+        Hallo <x id="INTERPOLATION" equiv-text="{{ data.Firstname }}"/>,<x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+        leider kannst du via Phonebook deine Daten nicht ändern. <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+        Falls du deine Daten ändern möchtest wende dich bitte an deine Human Resources Abteilung.
+      </target>
+        <note priority="1" from="description">Profile</note>
+        <note priority="1" from="meaning">User-InformationDialog</note>
+      </trans-unit><trans-unit id="Colleague Profile" datatype="html">
+        <source>
+      Is there something wrong with the profile? <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+      Please let your colleague know that the data is incorrect and that it can be changed through your Human Resources
+      department.
+    </source><target state="new">
+        Stimmt etwas auf dem Profil nicht? <x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br/>"/>
+        Lass es doch deinen Kollegen wissen, dass er seine Daten über das Human Resources ändern lassen kann. 
+      </target>
+        <note priority="1" from="description">Profile</note>
+        <note priority="1" from="meaning">User-InformationDialog</note>
+      </trans-unit><trans-unit id="Notify" datatype="html">
+        <source>Notify <x id="INTERPOLATION" equiv-text="{{ data.person.Firstname }}"/></source><target state="translated">
+      Benachrichtige <x id="INTERPOLATION" equiv-text="{{ data.person.Firstname }}"/>
+    </target>
+        <note priority="1" from="description">Profile</note>
+        <note priority="1" from="meaning">User-InformationDialog</note>
+      </trans-unit><trans-unit id="SendEmail" datatype="html">
+        <source>Send your colleague a direct message</source><target state="translated">Hey sende deinem Kollegen direkt eine Nachricht</target>
+        <note priority="1" from="description">Button</note>
+        <note priority="1" from="meaning">User-InformationDialog</note>
+      </trans-unit><trans-unit id="GotItButton" datatype="html">
+        <source>
+      Got it!
+    </source><target state="new">
```

Comment:
```suggestion
    </source><target state="translated">
```

Would be good if you do change this every time. Later we can check with this state if the translation is already done.

Reply:
We should move on to something better that will do it automatically... Something like transifex or else